package.json

```diff
@@ -4,13 +4,14 @@
   "scripts": {
     "codegen:rinkeby": "graph codegen -- subgraph-rinkeby.yaml",
     "build:rinkeby": "graph build -- subgraph-rinkeby.yaml",
-    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- vbstreetz/witswap-staking-rinkeby subgraph-rinkeby.yaml",
+    "deploy:rinkeby": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker-rinkeby subgraph-rinkeby.yaml",
```

Comment:
We don't need any of these rinkeby deployments etc

Reply:
Please delete

Reply:
"codegen:mainnet": "graph codegen -- subgraph-mainnet.yaml",
"build:mainnet": "graph build -- subgraph-mainnet.yaml",
"deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ -- joyfulmagician/taraswap_staker subgraph-mainnet.yaml",
"deploy:local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 taraswap_staker subgraph-mainnet.yaml",